.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,64 @@
+name: 💚 CI Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    tags:
+      - '*'
+
+jobs:
+  # Static test of the back directory
+  test_static_back:
+    if: ${{ github.event_name == 'pull_request' || github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 22.13.0
+      - name: Cache Node.js dependencies
+        uses: actions/cache@v3
+        with:
+          path: back/node_modules
+          key: ${{ runner.os }}-node-modules-back-${{ hashFiles('back/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-node-modules-back-
+      - name: Install dependencies and run linting
+        run: |
+          cd back
```

Comment:
suggestion: You can use a [working-directory: ./back](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/setting-a-default-shell-and-working-directory)

Reply:
I keep this comment for the later version :pray: